ui/src/actions/__tests__/search.test.js

```diff
@@ -52,34 +52,54 @@ describe('search - action creators', () => {
         {
           type: types.SEARCH_ERROR,
           payload: { namespace, error: { status: 'network' } },
-          meta: { redirectableError: true }, // TODO: should this be redirectableError?
+          meta: { redirectableError: true },
```

Comment:
@nooraangelva why did you change this test? we should add a new one not modify the existing ones.

Reply:
I did add a new one and this is the new one. 
The other test (old one) is [here](https://github.com/inspirehep/inspirehep/blob/fc65ac2313b1cc6c17354ac38891c9db565cff3b/ui/src/actions/__tests__/search.test.js#L61-L79) I did have to change the `redirectableError` to `false` since in config it is false.